### PR TITLE
Add yarn reset script

### PR DESCRIPTION
**Pull request by devinjameson**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "clean": "rm -rf node_modules && yarn",
+    "clean": "yarn cache clean && watchman watch-del-all && rm -rf node_modules && yarn && rm ios/Podfile.lock && yarn install:pod && react-native start --resetCache",
```
Comment (aledustet) [inline package.json]: not sure we want the start at the end but I have that alias almost exactly. With an addition to clean the android project and delete derived data

Comment (johnschoeman) [inline package.json]:
```suggestion
    "clean": "yarn cache clean && watchman watch-del-all && rm -rf node_modules && rm ios/Podfile.lock && yarn && yarn install:pod",
    "start": "react-native start --resetCache"
    "clean-and-start": "yarn clean && yarn start"
```
minor suggestion, but I find having `clean` initiate metro bundler to be counter intuitive

**Review by johnschoeman — APPROVED** [review]
🥇

File: package.json (hunk 1 above)
Comment (devinjameson) [reply package.json]: Should we add the Android stuff as well?

Comment (aledustet) [reply package.json]: It would be running `./gradlew clean` inside the android folder. I don't see why not.

Comment (juancruzgs) [reply package.json]: You could add `./android/gradlew -p ./android clean` for Android

**Review by aledustet — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -14,6 +14,7 @@
     "run-ios": "yarn install:pod && react-native run-ios --scheme \"BT_Development\" --configuration \"Debug-BT\" --simulator \"iPhone 11 (13.5)\"",
     "preinstall": "node -e \"if(process.env.npm_execpath.indexOf('yarn') === -1) throw new Error('You must use Yarn to install, not NPM')\"",
     "install:pod": "cd ios && bundle install --quiet && bundle exec pod install --silent",
+    "nuke": "yarn cache clean && watchman watch-del-all && rm -rf node_modules && yarn && ./android/gradlew -p ./android clean && rm ios/Podfile.lock && yarn install:pod",
```
Comment (johnschoeman) [inline package.json]:
```suggestion
    "reset": "yarn cache clean && watchman watch-del-all && rm -rf node_modules && yarn && ./android/gradlew -p ./android clean && rm ios/Podfile.lock && yarn install:pod",
```

**Review by johnschoeman — APPROVED** [review]
LGTM

File: package.json (hunk 2 above)
Comment (devinjameson) [reply package.json]: 👍

**Pull request merged**